O/oneAPI_Level_Zero/oneAPI_Level_Zero_Loader/bundled/patches/musl-no-deepbind.patch

```diff
@@ -0,0 +1,25 @@
+From 3a15f35ccae10e0e8d21acb2cdb2790b2955a5be Mon Sep 17 00:00:00 2001
```

Comment:
It'd be great to upstream this patch (I guess it'll need a glibc guard or something like that) :slightly_smiling_face:

Reply:
I'm not actually sure it's correct, I just did it to make sure it compiles :smile: But yeah, I have a large number of musl patches I really should push to various upstreams.